test/e2e/mcd_test.go

```diff
@@ -153,6 +154,129 @@ func TestKernelArguments(t *testing.T) {
 	require.Nil(t, err)
 }
 
+func TestNoReboot(t *testing.T) {
```

Comment:
why break up the 2 kernel tests and put this in between?

Reply:
Will move